### PR TITLE
enhancement/678 - removes wind vane from `CanvasController`

### LINKED ISSUE #678: Remove wind vane from CanvasController
~~Currently the wind vane, and supporting logic, lives within the `CanvasController`.  This should be abstracted to a standalone view component (like the `StripView`, etc) built with html/css or simple svg.~~

Remove wind vane concerns from `CanvasController`

### PR DESCRIPTION
**Pull request by n8rzz**
Resolves #678 

Removes wind vance from `CanvasController`

### CODE REVIEW

File: src/assets/scripts/client/canvas/CanvasController.js [hunk 1]
```diff
@@ -1452,95 +1452,9 @@ export default class CanvasController {
...
-        }
-
-        cc.save();
-        cc.translate(
-            -dot / 2 * sin(airportModel.wind.angle),
-            dot / 2 * cos(airportModel.wind.angle)
-        );
+        // left over from old `._drawWindVane()` method. removing this, or moving it to `._drawAirspaceAndRangeRings`
+        // or `._drawRunwayLabels()` causes a white circle to be drawn with the airport center at its origin
         cc.beginPath();
```
Comment (n8rzz) [inline src/assets/scripts/client/canvas/CanvasController.js]: why this still needs to be here is beyond me.  I can't for the life of me figure out why this still needs to be here.  Moving this to either method called before or after this results in the same white circle.

**Comment by felixscheffer** [comment]
I tried to fix some beginPath() related issues. Can somebody confirm that there are no drawing issues, please?

@n8rzz @erikquinn @Marcel510

**Comment by Marcel510** [comment]
@felixscheffer I took a look there aren't any real drawing issues I'd say, only when you play around with the strip bar there's sometimes a white flickering thing. Nothing severe at all from my perspective and I can't confirm it's due to the changes in this branch, just wanted to let you know. (Couldn't reproduce them in the master branch though.) 
I'll try to get a video up

**Comment by erikquinn** [comment]
Any problems with classic theme? One of the comments from the removed bits say `if removed, white lines appear on top of bottom half of lock case`.

**Comment by erikquinn** [comment]
@n8rzz Could you resolve the conflicts? I can't quite see for sure how they should be combined.

**Comment by n8rzz** [comment]
@erikquinn conflicts resolved and building now

**Pull request merged**